spec/fixtures/extra_countries/en.yml

```diff
@@ -0,0 +1,7 @@
+SU:
```

Comment:
Actually is this *and* `EXTRA_COUNTRIES` necessary? `EXTRA_COUNTRIES` can be populated from this or, in the cases where we need a file, it can be `YAML.dump`ed to this. 

What do you think?